docs/product-assembly_business-optimizer/src/main/asciidoc/main.adoc

```diff
@@ -15,43 +14,55 @@ As a business rules developer, you can use the {PLANNER} to find the optimal sol
 Use this document to set up and configure {PLANNER} functionality.
 
 .Prerequisite
-The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see _Getting started with decision services_.
-//@doc-link:Add link to Getting Started assembly above. 
+The *Employee Rostering* sample project has been created in {CENTRAL} to demonstrate Business Optimizer capabilities. To view this project, you must have {PRODUCT} installed. For more information about setting up and installing {PRODUCT} and {CENTRAL}, see {URL_GETTING_STARTED_DECISION_SERVICE}[{GETTING_STARTED_DECISION_SERVICE}].
```

Comment:
+The *Employee Rostering* sample project has been created in ....
>>
+The *Employee Rostering* sample project is created in ...